pkg/deviceshifu/deviceshifuhttp/deviceshifuhttp.go

```diff
@@ -96,7 +96,7 @@ func New(deviceShifuMetadata *deviceshifubase.DeviceShifuMetaData) (*DeviceShifu
 				mux.HandleFunc("/"+instruction, handler.commandHandleFunc())
 			}
 		default:
-			klog.Errorf("EdgeDevice protocol %v not supported in deviceShifu_http_http", protocol)
+			zlog.Errorf("EdgeDevice protocol %v not supported in deviceShifu_http_http", protocol)
```

Comment:
Let's use the `logger` name directly, no need to rename it to zlog, same for other occurrences

Reply:
ok